src/Compilers/Core/Portable/WellKnownMembers.cs

```diff
@@ -2929,6 +2929,36 @@ static WellKnownMembers()
                      (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Void,
                      (byte)SignatureTypeCode.Pointer, (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Void,
                      (byte)SignatureTypeCode.TypeHandle, (byte)SpecialType.System_Int32,
+
+                 // System_Span__get_Item
```

Comment:
Please test with Span/ReadOnlySpan implementations with bad Item signatures (type, ref-ness).

Reply:
for example - TestSpanValIndexer()

---
In reply to: [153593868](https://github.com/dotnet/roslyn/pull/23363#discussion_r153593868) [](ancestors = 153593868)